clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -408,6 +411,11 @@ public class VenicePushJob implements AutoCloseable {
    */
   public static final String TARGETED_REGION_PUSH_LIST = "targeted.region.push.list";
 
+  /**
+   * Config to enable post validation and consumption in VPJ when {@link #TARGETED_REGION_PUSH_ENABLED} is true.
+   */
+  public static final String POST_VALIDATION_CONSUMPTION_ENABLED = "post.validation.consumption.enabled";
```

Comment:
Do we need this ? what case we would not want to validate the push?

Reply:
By default, this config is true so we always validate. Just in case some customers would like to do a true single region push hence I provide this config.

Reply:
What would be the behavior if its false ? are we supposed to return early without doing rest of the colo push ?

Reply:
Yes. That would be a true single region push